### DISCUSSION TITLE
Add loaders

### DISCUSSION THREAD

**Discussion by cake03** [Homebrew/discussions#1608] [Everyday usage]
Make terminal loaders or else people who don't use their computer as much will think that Homebrew isn't working

**Comment by carlocab** [comment]
What is a terminal loader?

**Reply by gromgit** [reply]
Sounds like progress bars for long operations, but the only one I can think of is already covered by a PR: https://github.com/Homebrew/install/pull/547

**Reply by carlocab** [reply]
There is `HOMEBREW_VERBOSE_USING_DOTS`, but that only applies with `--verbose`.

**Reply by gromgit** [reply]
Today I learned how much of the `brew` man page I've actually read. :man_shrugging: :grin:
